spec/lib/hets/szs_parser_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe Hets::Prove::SZSParser do
+  let(:fixtures_dir) do
+    Rails.root.join('spec', 'fixtures', 'prover_outputs')
+  end
+
+  %w(Theorem CounterSatisfiable).each do |szs_status|
+    %w(MathServeBroker
+       SPASS
+       Vampire).each do |prover|
+      context "#{prover} on #{szs_status}" do
+        let(:output) { File.read(fixtures_dir.join(szs_status, prover)) }
+
+        it "returns nil" do
+          expect(Hets::Prove::SZSParser.new(prover, output).call).to be(nil)
```

Comment:
I'm not sure. First, we should check if we (Hets) can start SPASS (and Vampire) with other parameters to give us the SZS output like the other provers. If that's not possible, I will parse SPASS's output like you suggested.